components/table/demo/row-selection-and-operation.md

```diff
@@ -1,8 +1,16 @@
 ---
 order: 2
-title: 选择和操作
+title:
+  en-US: Selection and action
+  zh-CN: 选择和操作
 ---
 
+## en-US
+
+Make actions and clear selection after select some rows by using `rowSelection.selectedRowKeys` to control selected rows.
```

Comment:
To perform operations and clear selections after selecting some rows, use `rowSelection.selectedRowKeys`